tasks/install.yml

```diff
@@ -39,7 +41,9 @@
   with_items:
     - "{{ restic_cron_stdout_file }}"
     - "{{ restic_cron_stderr_file }}"
-  when: item != None
+  when:
+    - item != None
```

Comment:
Shouldn't this be "is not match"? Because you don't want to chmod stuff under /dev/?

Reply:
Yes, of course... I noticed this just after pushing :man_facepalming: